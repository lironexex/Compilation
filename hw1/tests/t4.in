// IDs:
a
Z
_compi
_Z123ab
_cC1aA
E1234S
_one_two_three_    // the last '_' char by itself is not a legal ID
