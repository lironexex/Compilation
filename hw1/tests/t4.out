1 COMMENT 1
2 ID a
3 ID Z
4 ID _compi
5 ID _Z123ab
6 ID _cC1aA
7 ID E1234S
8 ID _one
8 ID _two
8 ID _three
Error _
